android/app/src/main/res/layout/info.xml

```diff
@@ -14,215 +15,101 @@
+        android:textColor="?attr/colorOnPrimary"
+        app:layout_constraintEnd_toStartOf="@id/toolbar_text_space_place_holder" />
 
-    <TextView
-        android:id="@+id/after_8"
-        style="@style/TextAppearance.AppCompat.Medium.Inverse"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="open"
-        android:textAppearance="?textAppearanceBody1"
-        android:textColor="?attr/colorOnPrimary" />
 
     <TextView
-        android:id="@+id/after_9"
-        style="@style/TextAppearance.AppCompat.Medium.Inverse"
+        android:id="@+id/toolbar_text_space_place_holder"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="source"
+        android:text="Roboto"
         android:textAppearance="?textAppearanceBody1"
-        android:textColor="?attr/colorOnPrimary" />
+        android:visibility="invisible"
+        app:layout_constraintHorizontal_bias="0.45" />
 
     <TextView
-        android:id="@+id/after_10"
-        style="@style/TextAppearance.AppCompat.Medium.Inverse"
+        android:id="@+id/toolbar_text"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="libraries."
+        android:text="Roboto"
         android:textAppearance="?textAppearanceBody1"
-        android:textColor="?attr/colorOnPrimary" />
-
-    <Button
-        android:id="@+id/contributor_invite"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        app:layout_constraintStart_toStartOf="@+id/toolbar"
-        app:layout_constraintEnd_toEndOf="@+id/toolbar"
-        app:layout_constraintTop_toBottomOf="@+id/after_10"
-        app:layout_constraintBottom_toBottomOf="@+id/toolbar"
-        android:text="@string/contributor_invite"
         android:textColor="?attr/colorOnPrimary"
-        style="@style/Widget.MaterialComponents.Button.TextButton" />
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="@id/inset" />
 
     <TextView
-        android:id="@+id/hider"
+        android:id="@+id/end_text"
         android:layout_width="0dp"
-        android:layout_height="0dp"
-        android:alpha="0"
-        android:background="@color/colorPrimary"
-        app:layout_constraintEnd_toEndOf="@+id/toolbar"
-        app:layout_constraintStart_toStartOf="@+id/toolbar"
-        app:layout_constraintTop_toTopOf="@+id/toolbar"
-        app:layout_constraintBottom_toBottomOf="@+id/toolbar"/>
-
-    <TextView
-        android:id="@+id/title"
-        style="@style/TextAppearance.AppCompat.Medium.Inverse"
-        android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:text="Roboto"
-        android:gravity="start|center_vertical"
+        android:text=" app is open souce,"
```

Comment:
These were intentionally broken up so the text would still flow on smaller devices (and larger ones). Does this mitigate that in some way?